fix(Alert): update alert a11y to address title elements (#3348)
patternfly/patternfly-org: packages/v4/patternfly-docs/content/accessibility/alert/alert.md

  <ListItem>
    <Checkbox id="alert-a11y-checkbox-1" label="If an alert will dynamically appear or update, it is rendered inside of an alert group component." />
  </ListItem>
  <ListItem>

>     <Checkbox id="alert-a11y-checkbox-2" label="If an alert is using a heading level for its title, the heading level should fit the page heading level order so that heading levels are not skipped." />

Reviewer: ```suggestion
    <Checkbox id="alert-a11y-checkbox-2" label="If an alert is using a heading element for its title, the heading level fits the page heading level order so that heading levels are not skipped." />
```

  </ListItem>
  <ListItem>

>     <Checkbox id="alert-a11y-checkbox-2" label="If an alert's title is not describing any content in a description below it, the title should be wrapped in a non-heading element such as a 'span' or a 'div'." />

Reviewer: ```suggestion
    <Checkbox id="alert-a11y-checkbox-2" label="If an alert does not have any description below its title, the title is wrapped in a non-heading element such as a 'span' or a 'div'." />
```

I can't think of a good way to word one at the moment, but a description here quickly summarizing the "why" would be great. Best I can come up with is, "Without a description, a heading element in the title may cause other, unrelated content that follows seem related when navigating the page via rotor menus." Not a blocker if we don't get a description in, though.
